src/parachain/loans.ts

```diff
@@ -304,7 +313,45 @@ export class DefaultLoansAPI implements LoansAPI {
         };
     }
 
-    async getLendPositionsOfAccount(accountId: AccountId): Promise<Array<LendPosition>> {
+    _calculateEarnedDebt(borrowedAmount: Big, snapshotBorrowIndex: Big, currentBorrowIndex: Big): Big {
```

Comment:
Since debt is a liability rather than an asset, "earned" isn't the best wording here. I suggest renaming this to `_calculateAccumulatedDebt`.

Reply:
good suggestion, thanks!